examples/dmm/polyphonic_data_loader.py

```diff
@@ -94,6 +94,7 @@ def load_data(dataset):
         for k, v in dset.items():
             sequences = v["sequences"]
             dset[k]["sequences"] = pad_sequence(sequences, batch_first=True).type(torch.Tensor)
+            dset[k]["sequence_lengths"] = v["sequence_lengths"].to(device=torch.Tensor().device)
```

Comment:
Fails non-deterministically for the HMM example using `--cuda` otherwise, because the sequence lengths are used to define the mask (I am not sure why this passes sometimes). cc. @martinjankowiak